gnocchi/opts.py

```diff
@@ -149,12 +149,6 @@ def list_opts():
                        help="How many seconds to wait between "
                        "metric ingestion reporting. Set value to -1 to "
                        "disable reporting"),
-            cfg.IntOpt('metric_cleanup_delay',
```

Comment:
we should probably keep this? part of the issue is that it does nothing on cleanup also as there's little to cleanup. if we remove this, it's checking to do nothing even more often.

Reply:
Not sure I'm smart enough to do that. I'll keep a look :)